### PR TITLE
fix: font security

### PR DESCRIPTION
**Pull request by catheraaine**
On the live, secure sites, fonts do not currently load. Changed `font-face` rules to use a relative url so that the fonts will load whether the user is at `http` or `https`.

Not entirely sure how to validate this, all the fonts still load locally and the sources tab shows the right url.

### CODE REVIEW

File: Fonts/493470/753D1281059CB826E.css [hunk 1]
```diff
@@ -16,4 +16,4 @@
...
...rl('//apprentices.seesparkbox.com/Fonts/493470/2E2221D311E64A3CA.eot?#hco') format('embedded-opentype'); font-style:italic; font-weight:500; } @font-face { font-family:"Archer A"; src: url('//apprentices.seesparkbox.com/Fonts/493470/C1763FE6CCDAC4FA7.eot'); src: url('//apprentices.seesparkbox.com/Fonts/493470/C1763FE6CCDAC4FA7.eot?#hco') format('embedded-opentype'); font-style:normal; font-weight:800; } @font-face { font-family:"Archer 8r"; src: url('//apprentices.seesparkbox.com/Fonts/493470/C1763FE6CCDAC4FA7.eot'); src: url('//apprentices.seesparkbox.com/Fonts/493470/C1763FE6CCDAC4FA7.eot?#hco') format('embedded-opentype'); font-style:normal; font-weight:800; } @font-face { font-family:"Archer A"; src: url('//apprentices.seesparkbox.com/Fonts/493470/7FE003FA659397C49.eot'); src: url('//apprentices.seesparkbox.com/Fonts/493470/7FE003FA659397C49.eot?#hco') format('embedded-opentype'); font-style:italic; font-weight:800; } @font-face { font-family:"Archer 8i"; src: url('//apprentices.seesparkbox.com/Fonts/493470/7FE003FA659397C49.eot'); src: url('//apprentices.seesparkbox.com/Fonts/493470/7FE003FA659397C49.eot?#hco') format('embedded-opentype'); font-style:italic; font-weight:800; }
```
Comment (nrambeck) [inline Fonts/493470/753D1281059CB826E.css]: To make this more robust/portable, we could actually just use a relative path only and remove the domain altogether.

**Comment by nrambeck** [comment]
So I looked into Typography.com settings. We can change the location it will redirect to from http to https, but it will change which folder it expects the fonts in (sigh). So we need to coordinate that changes with these changes. After making a change to the settings, take note of the new directory location and do a `git mv` on our fonts to the new folder location.

image

**Comment by catheraaine** [comment]
@nrambeck I followed the instructions on Typograpy.com and I _know_ I've done this for other projects before but it's one of those things that fall out of my head as soon as I don't have to work on it anymore.

**Comment by catheraaine** [comment]
also going to force ssl on this pr, but i need to drop a little js into the project on monday

**Comment by catheraaine** [comment]
Whenever you're ready @nrambeck

**Pull request merged**